addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SpringActuatorScanRule.java

```diff
@@ -196,15 +197,30 @@ private HttpMessage sendActuatorRequest(String encodingType, String actuatorEndp
         return null;
     }
 
-    private void raiseAlert(HttpMessage msg, int confidence, int risk) {
-        newAlert()
-                .setRisk(risk)
-                .setConfidence(confidence)
+    private AlertBuilder createAlert(HttpMessage msg) {
```

Comment:
```suggestion
    private AlertBuilder createAlert(String evidence) {
```
(`evidence` or `responseBody`)

Reply:
Hello @thc202, could you specify which parts also need to be updated? Im a bit confused where and what should we change.

Reply:
The calls to this method need to be updated to pass the evidence instead of the message.

Reply:
Hello @thc202 i changed the message to the evidence on the calls of the method, if there is any other change needed on this alert, let us know. Thank you for your help!